gatekeeper/client/client.go

```diff
@@ -29,20 +32,46 @@ type Client struct {
 }
 
 // NewClient returns a new client to a Gatekeeper host that can bootstrap this machine
-func NewClient(host string) *Client {
+func NewClient(host, caFile string) (*Client, error) {
 	if !strings.HasSuffix(host, "/") {
 		host += "/"
 	}
+
+	caPool, err := x509.SystemCertPool()
+	if err != nil {
```

Comment:
Should we log the error? As in, "Could not load System Cert Pool, error: %s, creating custom certificate pool"?

Also, thinking about this more, I think this would be fatal if a user didn't specify the `--ca` flag?